src/test/resources/munit/body/form/form-body-api-oas30.json

```diff
@@ -305,6 +305,39 @@
             "required": true
           }
         }
+      },
+      "/shipments/agentDocument": {
+        "post": {
+          "summary": "Test",
+          "description": "Test",
+          "operationId": "addTest",
+          "requestBody": {
+            "content": {
+              "multipart/form-data": {
+                "schema": {
+                  "required": [
+                    "partnerId"
+                  ],
+                  "properties": {
+                    "partnerId": {
+                      "type": "array",
+                      "description": "partner code",
+                      "items": {
+                        "type": "string"
```

Comment:
As a parameter of type string will validate successfully any value, I would try with a different parameter type, like an integer number. This way we can document the positive and negative validations.
In other words, type being a string would allow any value with commas inside, while an integer not, so an array of the following strings will be valid:
```
- "123"
- "456,789" --> valid string literal
```
while an array of the following integers won't:
```
- 123
- 456,789 --> invalid integer
```